quartz/components/scripts/search.inline.ts

```diff
@@ -222,16 +222,20 @@ document.addEventListener("nav", async (e: unknown) => {
 
   const resultToHTML = ({ slug, title, content, tags }: Item) => {
     const htmlTags = tags.length > 0 ? `<ul>${tags.join("")}</ul>` : ``
-    const button = document.createElement("button")
-    button.classList.add("result-card")
-    button.id = slug
-    button.innerHTML = `<h3>${title}</h3>${htmlTags}<p>${content}</p>`
-    button.addEventListener("click", () => {
+    const itemTile = document.createElement("a")
+    itemTile.classList.add("result-card")
+    itemTile.id = slug
+    itemTile.innerHTML = `<h3><a href="${new URL(
```

Comment:
now that `itemTile` is `a`, let's set the `itemTile.href` so we don't need to nest `a` here again

Reply:
addressed. PTAL.